### PR TITLE
Add com.geeks3d.furmark

### PR DESCRIPTION
**Pull request by hannescam**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  
Furmark is a benchmarking and stress-testing utility for GPUs it supports OpenGL and Vulkan

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link: contacted via e-mail**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.geeks3d.furmark.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        <ul>
+           <li>fixed bugs in the Vulkan rendering.</li>
+           <li>improved Linux support.</li>
+           <li>GUI settings are now saved in settings.lua.</li>
+           <li>added BFC (back face culling) checkbox in the GUI.</li>
+           <li>added OpenGL 2.1, 3.0 and 3.1 support.</li>
+           <li>added core hotspot temperature for NVIDIA GPUs.</li>
+           <li>added a tooltip over preset buttons when they are disabled.</li>
+           <li>utilities buttons (cpu burner, gpu shark, gpu-z) are disabled if they are not present.</li>
+           <li>new commercial names:
```
Comment (bbhtt) [inline com.geeks3d.furmark.metainfo.xml:106]: Add this line in a p tag then add the items below as li.

File: com.geeks3d.furmark.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+command: FurMark_GUI
+
+modules:
+
+  - name: libglu
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.freedesktop.org/mesa/glu
+        branch: master
```
Comment (bbhtt) [inline com.geeks3d.furmark.yml:22]: This should be a tag or a commit, never a branch.

File: com.geeks3d.furmark.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      - chmod +x /app/furmark/FurMark_GUI
+      - ln -s /app/furmark/FurMark_GUI /app/bin/FurMark_GUI
+      - ffmpeg -i furmark-icon.png -vf scale=512:512 furmark-icon-scaled.png
+      - install -Dm644 furmark-icon-scaled.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png # Install the app icon
+      - install -Dm644 com.geeks3d.furmark.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop # Install the app menu entry .desktop file
+      - install -Dm644 com.geeks3d.furmark.metainfo.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.metainfo.xml # Install the app metadata
+
+    sources:
+      - type: archive
+        url: https://gpumagick.com/downloads/files/2024/furmark2/FurMark_2.3.0.0_linux64.zip
```
Comment (bbhtt) [inline com.geeks3d.furmark.yml:39]: This is x86_64 only, you are building for both arches. See https://docs.flathub.org/docs/for-app-authors/requirements#flathubjson

File: com.geeks3d.furmark.desktop
Comment (bbhtt) [inline com.geeks3d.furmark.desktop:1]: Why is desktop file not in the tarball?

**Comment by hannescam** [comment]
bot, build com.geeks3d.furmark

**Comment by hannescam** [comment]
bot, build com.geeks3d.furmark

File: com.geeks3d.furmark.desktop
Comment (hfiguiere) [inline com.geeks3d.furmark.desktop:1]: why isn't this file upstream?

**Comment by hannescam** [comment]
@hfiguiere could you please review my response

**Pull request closed without merging**